tensorflow_probability/python/stats/quantiles.py

```diff
@@ -899,7 +896,8 @@ def _move_dims_to_flat_end(x, axis, x_ndims, right_end=True):
 
 
 def _sort_tensor(tensor):
-  """Use `top_k` to sort a `Tensor` along the last dimension."""
-  sorted_, _ = tf.math.top_k(tensor, k=tf.shape(tensor)[-1])
+  """Use `sort` to sort a `Tensor` in ascending order along
+     the last dimension."""
+  sorted_ = tf.sort(tensor, axis=-1, direction='ASCENDING')
   tensorshape_util.set_shape(sorted_, tensor.shape)
```

Comment:
i think the set_shape is no longer needed, since tf.sort will not change the shape (top_k would have the potential to, although we were abusing it slightly by setting k to the size of the input).

Reply:
@csuter As this can then be done in 1 line now
```python
return tf.sort(tensor, axis=-1, direction='ASCENDING')
```
and there is only 1 function call to `_sort_tensor` should I just remove `_sort_tensor` altogether and instead have
```python
sorted_y = tf.sort(y, axis=-1, direction='ASCENDING')
```
?

Reply:
yes, that sounds great. thanks!

Reply:
Done.